rllib/utils/replay_buffers/replay_buffer.py

```diff
@@ -17,35 +19,47 @@
 logger = logging.getLogger(__name__)
 
 
+@ExperimentalAPI
+class StorageUnit(Enum):
+    TIMESTEPS = 0
+    SEQUENCES = 1
+    EPISODES = 2
+
+
 @ExperimentalAPI
 class ReplayBuffer:
-    def __init__(
-        self, capacity: int = 10000, storage_unit: str = "timesteps", **kwargs
-    ):
+    def __init__(self, capacity: int = 10000,
+                 storage_unit: str ="timesteps", **kwargs):
         """Initializes a ReplayBuffer instance.
 
         Args:
             capacity: Max number of timesteps to store in the FIFO
                 buffer. After reaching this number, older samples will be
                 dropped to make space for new ones.
-            storage_unit: Either 'sequences' or 'timesteps'. Specifies how
-                experiences are stored.
+            storage_unit: Either 'timesteps', `sequences` or
+                `episodes`. Specifies how experiences are stored.
             **kwargs: Forward compatibility kwargs.
         """
 
-        if storage_unit == "timesteps":
-            self._store_as_sequences = False
-        elif storage_unit == "sequences":
-            self._store_as_sequences = True
+        if storage_unit in ["timesteps", StorageUnit.TIMESTEPS]:
```

Comment:
why keep the hardcoded string names?

Reply:
Because it gives the user the flexibility to use the string as an argument.
Users might want to specify a buffer using strings, because it is brief and readable.

I just chose to enumerate here because I felt like it's prone to errors writing out "timesteps" or other strings dozens of times.

Should I still change it?

Reply:
Oh, I think we thought that `StorageUnit.TIMESTEPS` is actually a string ("timesteps"). :D
I see it's an enum? Should we just make these strings?

```
STORAGE_UNIT_TIMESTEPS = "timesteps"
...
```

We don't really use enums anywhere else in RLlib.